dask_cuda/cuda_worker.py

```diff
@@ -103,11 +103,15 @@ def del_pid_file():
             atexit.register(del_pid_file)
 
         if resources:
-            resources = resources.replace(",", " ").split()
+            resources = resources.replace(",", " ").replace("'", "").split()
             resources = dict(pair.split("=") for pair in resources)
-            resources = valmap(float, resources)
+            gpu_resources = valmap(int, itemfilter(lambda x: x != "GPU", resources))
+            resources = valmap(float, itemfilter(lambda x: x == "GPU", resources))
```

Comment:
@jacobtomlinson you have written this section originally, I presume mapping values to `float` was done to support a definition such as `"MEMORY"` but since I don't see any tests or any other explicit mention in Dask-CUDA, could you comment if that's right and whether you can think of more robust ways for us to handle types here other than what I wrote above as _"GPU"_ or _NOT "GPU"_?

Reply:
Honestly I don't remember the reasoning. I think dask handles these values as floats, as you say it's to support values like `MEMORY` or other arbitrary quantities.